beta-4-setup.md

```diff
@@ -1237,20 +1239,11 @@ interface, unlike most containers that listen only on private IPs. The router
 proxies external requests for route names to the IPs of actual pods identified
 by the service associated with the route.
 
-OpenShift's admin command set enables you to deploy router pods automatically.
-As the `root` user, try running it with no options and you will see that
-some options are needed to create the router:
+AE's admin command set enables you to deploy router pods automatically.
```

Comment:
Oops, I'll fix it. Good catch :)

AE - Atomic Enterprise ... I'm working on a stripped down tutorial for it.